### PR TITLE
[TH2-3886] th2-infra > Pass infra-mgr key as secret value

### PR DESCRIPTION
**Pull request by sabamushtashvili**

### CODE REVIEW

**Review by Andrei-Shulika — CHANGES_REQUESTED** [review]
@sabamushtashvili 
Need to change this section. Private key is required at installation.
This comment need to be deleted according to comment above.
`# required if http(s) access to gitlab/github repositories is used`

**Comment by bop0hz** [comment]
We also must update comments in values.yaml to more meaningful and compliant to doc generator

File: chart/templates/configs/secret-infra-mgr.yaml [hunk 1]
```diff
@@ -1,9 +1,25 @@
...
 metadata:
-  name: infra-mgr
+  name: {{ .Values.infraMgr.git.secretName }}
+  annotations:
+    meta.helm.sh/release-name: "th2-infra"
+    meta.helm.sh/release-namespace: "service"
+  {{- include "common-annotations" . | nindent 4  }}
+  labels:
+    {{- include "th2-labels" . | nindent 4 }}
+    app.kubernetes.io/managed-by: "Helm"
```
Comment (bop0hz) [inline chart/templates/configs/secret-infra-mgr.yaml:11]: must not contain helm labels and annotations

File: chart/templates/configs/secret-infra-mgr.yaml [hunk 2]
```diff
@@ -1,9 +1,25 @@
-{{- if .Values.infraMgr.git.privateKey -}}
 apiVersion: v1
 kind: Secret
 metadata:
-  name: infra-mgr
+  name: {{ .Values.infraMgr.git.secretName }}
```
Comment (bop0hz) [inline chart/templates/configs/secret-infra-mgr.yaml]: do not set value for name, just hardcode it

File: chart/tests/secret-infra-mgr_test.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      - equal:
+          path: data.httpAuthPassword
+          value: cGFzc3dvcmQ=
+  - it: ssh
+    set:
+      infraMgr:
+        git:
+          httpAuthUsername: "th2"
+          httpAuthPassword: "password"
+          privateKey: "key"
```
Comment (bop0hz) [inline chart/tests/secret-infra-mgr_test.yaml:39]: where is repository value?

File: README.md [hunk 4]
```diff
@@ -227,7 +232,8 @@ rabbitmq:
 
 # required if http(s) access to gitlab/github repositories is used
 #infraMgr:
-#  git:
+# git:
```
Comment (bop0hz) [inline README.md:235]: please add one space character

File: chart/tests/secret-infra-mgr_test.yaml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+tests:
+  - it: Name
+    set:
+      infraMgr:
+        git:
+          secretName: "infra-mgr"
+    asserts:
+      - equal:
+          path: metadata.name
+          value: infra-mgr
```
Comment (bop0hz) [inline chart/tests/secret-infra-mgr_test.yaml:15]: redundant

**Review by bop0hz — APPROVED** [review]

**Pull request merged**